### ISSUE TITLE
No activity

### ISSUE THREAD

**Issue by anthonyholmes** [AtomLinter/linter-php#111]
linter-php v1.1.6
php v7.0.0
OSX El Captian 10.11.2
Atom 1.3.1

The linter shows no activity. Works fine with linter-php v1.1.0

Running `apm test` from directory yields

````
Window load time: 466ms
...F..

The php -l provider for Linter
  checks bad.php and
    it verifies that message
      Expected [ [ 1, 0 ], [ 1, 6 ] ] to equal [ [ 1, 0 ], [ 1, 5 ] ].
        Error: Expected [ [ 1, 0 ], [ 1, 6 ] ] to equal [ [ 1, 0 ], [ 1, 5 ] ].
        at /Users/{USERNAME}/.atom/packages/linter-php/spec/linter-php-spec.js:54:37


Finished in 0.532 seconds
6 tests, 14 assertions, 1 failure, 0 skipped

Tests failed
````

**Comment by Arcanemagus** [comment]
The test failing has already been fixed on master, just haven't released an update since no client side code has been changed.

You might be hitting the `activationHooks` bug though, have you tried closing the currently open file and opening it again?

**Comment by anthonyholmes** [comment]
Closing the file and reopening worked like a charm! Thanks for the quick response

**Comment by Arcanemagus** [comment]
It looks like they should have a fix for it in v1.4.0, marking this as closed for now.
